src/WebSDL/settings/base.py

```diff
@@ -152,11 +154,14 @@
 
 RECAPTCHA_VERIFY_URL = "https://www.google.com/recaptcha/api/siteverify"
 
+EMAIL_SENDER = data['password_email_sender'] if 'password_email_sender' in data else '',
 
-EMAIL_SENDER = data['email_sender'] if 'email_sender' in data else '',
+NOTIFY_EMAIL = data['notify_email_sender'] if 'notify_email_sender' in data else ''
```

Comment:
In the settings file, we call it 'notify_email_sender'. It's only semantic, but we should change it to 'no_reply_email_sender' or something of that ilk.

Reply:
Noted. I'll add this to a list of things of issues to complete before the leaf pack network integration, in the meantime I'll change it in the server's settings.json file.